src/content/fr/2019/resource-hints.md

```diff
@@ -0,0 +1,250 @@
+
+### `preload`
+
+[`preload`](https://medium.com/reloading/preload-prefetch-and-priorities-in-chrome-776165961bbf) initie une requête au plus tôt. Il permet de charger des ressources importantes plutôt que d'attendre que l'analyseur les découvre plus tardivement. Par exemple, si une image importante ne peut être découverte qu'une fois que le navigateur a reçu et analysé la feuille de style, il est peut-être judicieux de précharger l'image.
+
+### `prefetch`
+
+[`prefetch`](https://calendar.perfplanet.com/2018/all-about-prefetching/) lance une requête de faible priorité. C'est utile pour charger les ressources qui seront utilisées lors du chargement de la page suivante (plutôt que de la page actuelle). Une utilisation courante de `prefetch` est le chargement de ressources dont l'application "prédit" qu'elles seront utilisées lors du chargement de la page suivante. Ces prédictions peuvent être basées sur des signaux tels que le mouvement de la souris de l'utilisateur ou des scénarios&#8239;/&#8239;parcours utilisateurs courants.
+
+## Syntaxe
+
+97 % des usages d'indices de ressources sont spécifiés par le tag [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). Par exemple, `<link rel="prefetch" href="shopping-cart.js">`.
+
+Seuls 3 % des indices de ressources proviennent [d'en-têtes HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) permettant de les spécifier. Par exemple, `Link: <https://example.com/shopping-cart.js>; rel=prefetch`.
+
+Comme l'utilisation des indices de ressources dans les en-têtes HTTP est très faible, le reste de ce chapitre se concentrera uniquement sur l'analyse de l'utilisation des indices de ressources passant par la balise `<link>`. Cependant, il convient de noter que dans les années à venir, l'utilisation des indices de ressources dans les en-têtes HTTP pourrait augmenter avec l'adoption de [HTTP/2 Push](./http2#http2-push). En effet, HTTP/2 Push réutilise l'en-tête HTTP de préchargement `Link` comme un signal permettant de pousser des ressources.
+
+## Les indices de ressources
+
+<aside class="note">Note: il n'y a pas de différence notable entre l'usage des indices de ressources sur les pages web destinées aux ordinateurs ou périphériques mobiles. C'est pourquoi, par souci de concision, ce chapitre n'inclut que les statistiques relatives aux mobiles.</aside>
+
+<figure>
+  <table>
+    <tr>
+     <th>Resource Hint</th>
+     <th>Utilisation (pourcentage de sites)</th>
+    </tr>
+    <tr>
+     <td><code>dns-prefetch</code>
+     </td>
+     <td>29 %
```

Comment:
Don't think these all need `&nbsp;` as you normally do?

```suggestion
     <td>29&nbsp;%
```

Similar with the big numbers?

Reply:
You're right, I've used invisible non-breaking spaces but no one can see them and it would be better with visible ones. I'll commit a future commit with this everywhere.

Reply:
Resolved by 8f874ca

Reply:
Out of interest what are "invisible non-breaking spaces"? Do you mean you originally used ASCII 160 instead of ASCII 32? So looked the same to me but was actually a non-breaking space?

Reply:
When typing, you can insert either spaces, non-break spaces or HTML-encode non-break spaces (screenshot below). I use the @nhoizey's Gremlins extension for VS Code to visualize the invisible special characters.

![Capture d’écran 2020-01-30 à 11 53 53](https://user-images.githubusercontent.com/284742/73443665-6c839580-4357-11ea-90bf-091424fe8238.png)

Reply:
Cool. Which uses U+00a0 which is 160 in real numbers, so yes that is what you were doing 😊

Reply:
Might be "fine unsecable space", but it is not rendered by Safari ^^